application/src/main/java/org/togetherjava/tjbot/commands/UserInteractorPrefix.java

```diff
@@ -0,0 +1,101 @@
+package org.togetherjava.tjbot.commands;
+
+import org.jetbrains.annotations.Contract;
+
+/**
+ * Contains the prefixes for the UserInteractor's.
+ * <p>
+ * This is used for separate interactors with the same name, by command type (and possibly more in
+ * the future). Our system doesn't allow multiple interactors with the same name, while having a
+ * slash-command and a message-context-command with the same name can be really useful.
+ */
+public enum UserInteractorPrefix {
+    /*
+     * Implementations that are none of the following have no dedicated prefix
+     */
+
+    SLASH_COMMAND(SlashCommand.class, "s-"),
+    MESSAGE_CONTEXT_COMMAND(MessageContextCommand.class, "mc-"),
+    USER_CONTEXT_COMMAND(UserContextCommand.class, "uc-");
+
+
+    private final Class<? extends UserInteractor> classType;
+    private final String prefix;
+
+    UserInteractorPrefix(Class<? extends UserInteractor> classType, final String prefix) {
+        this.classType = classType;
+        this.prefix = prefix;
+    }
+
+    /**
+     * The prefix for the command
+     *
+     * @return the command's prefix
+     */
+    public String getPrefix() {
+        return prefix;
+    }
+    /**
```

Comment:
no empty line between methods
```suggestion
    }

    /**
```